Testing/functional/tests/cypress/integration/ui/covid19/publicCovidTest.js

```diff
@@ -137,102 +130,103 @@ describe("Public User - Covid19 Test Result Page", () => {
 
         // Test Future Year does not exist
         cy.log("Testing Future Year does not exist.");
-        selectNotExist(dobYearSelector, nextYear.toString());
-        selectNotExist(collectionDateYearSelector, nextYear.toString());
+        selectShouldNotContain(dobYearSelector, nextYear.toString());
+        selectShouldNotContain(collectionDateYearSelector, nextYear.toString());
 
         // Test Current Year
-        select(dobYearSelector, year.toString());
-        select(collectionDateYearSelector, year.toString());
+        selectOption(dobYearSelector, year.toString());
+        selectOption(collectionDateYearSelector, year.toString());
```

Comment:
@MikeLyttle I am just wondering what is our naming conventions. based on this article (although it is unit test). instead of selectShouldNotContain.. it can be Should_ThrowException_When_NotExist? https://dzone.com/articles/7-popular-unit-test-naming.

Reply:
I don't think we've discussed naming conventions for the functional tests.  For selectShouldNotContain, I tried to model it on the phrasing used by Cypress.